pgcli/main.py

```diff
@@ -286,12 +287,15 @@ def prompt_tokens(cli):
         with self._completer_lock:
             buf = PGBuffer(always_multiline=self.multi_line, completer=self.completer,
                     history=FileHistory(os.path.expanduser(history_file)),
-                    complete_while_typing=Always())
+                    complete_while_typing=Always(),
+                    accept_action=AcceptAction.RETURN_DOCUMENT)
```

Comment:
When pressing ``Enter``, the "accept action" of a buffer is executed. In the case that the search-buffer has the focus, enter causes it to go the the match in the previous buffer. When the system buffer has the focus, the accept action will execute the system command. For the default buffer you usually want it to return the content. The default here also changed to "do nothing", because it was very confusing in some other cases.